### PR TITLE
[MB-16218] service request documents uploader

### PR DESCRIPTION
**Pull request by pearl-truss**
## Summary

This PR addresses 3 things

- Creates an uploader to handle `ServiceRequestDocuments`
- Adds factory functions for `ServiceRequestDocument` and `ServiceRequestDocumentUpload` that are needed for server test
- Added a delete function to the `ServiceRequestDocumentUpload` model, very similar to the one on the `PrimeUpload` model. One thing to note is that we're trying to move away from methods like these on the model and instead use service objects. While I think that was out of scope for this ticket given time constraints, I would like to move this into the existing `mtoserviceitem` service object as part of the work for https://dp3.atlassian.net/browse/MB-16192 if there's time

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [x] Have the Jira acceptance criteria been met for this change?

### CODE REVIEW

File: pkg/factory/service_request_document_upload_factory_test.go [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+// 			{
+// 				Model: models.Upload{},
+// 				ExtendedParams: &ServiceRequestDocumentUploadExtendedParams{
+// 					ServiceRequestDocumentUploader: uploader,
+// 				},
+// 			},
+// 		}, nil)
+// 	})
+
+// })
```
Comment (YanZ777) [inline pkg/factory/service_request_document_upload_factory_test.go:289]: **question:** Was this commented out code still meant to be here?

Comment (YanZ777) [inline pkg/factory/service_request_document_upload_factory_test.go:289]: **question:** Was this commented out code still meant to be here?

**Review by YanZ777 — APPROVED** [review]
Just a question about some commented code I saw, but other than that, looks good to me!

**Comment by pearl-truss** [comment]
> Just a question about some commented code I saw, but other than that, looks good to me!

Thanks, I forgot to remove this!

**Pull request merged**